full/src/test/java/apoc/load/LoadDirectoryTest.java

```diff
@@ -39,12 +39,12 @@ public static void setUp() throws Exception {
         // create temp files and subfolder
         temporaryFolder.newFile("Foo.csv");
         temporaryFolder.newFile("Bar.csv");
-        temporaryFolder.newFile("Baz.xls");
+        temporaryFolder.newFile("Baz.xsl");
         temporaryFolder.newFolder(IMPORT_DIR + File.separator + SUBFOLDER);
         temporaryFolder.newFile(IMPORT_DIR + File.separator + "TestCsv1.csv");
         temporaryFolder.newFile(IMPORT_DIR + File.separator + "TestCsv2.csv");
         temporaryFolder.newFile(IMPORT_DIR + File.separator + "TestCsv3.csv");
-        temporaryFolder.newFile(IMPORT_DIR + File.separator + "TestXls1.xsl");
+        temporaryFolder.newFile(IMPORT_DIR + File.separator + "Testxsl1.xsl");
```

Comment:
not that it matters too much, but I think `xls` was actually the correct extension